src/qt/peertablemodel.cpp

```diff
@@ -65,12 +65,19 @@ QVariant PeerTableModel::data(const QModelIndex& index, int role) const
         return QVariant();
 
     CNodeCombinedStats *rec = static_cast<CNodeCombinedStats*>(index.internalPointer());
-
+    const auto time_now{GetTime<std::chrono::seconds>()};
```

Comment:
0af493d include chrono
```diff
--- a/src/qt/peertablemodel.cpp
+++ b/src/qt/peertablemodel.cpp
@@ -9,11 +9,14 @@
 
 #include <interfaces/node.h>
 
+#include <chrono>
 #include <utility>
```

Reply:
applied in commit 4cacb8c0463307dfc02f077af5df2f79974ff9d2